modules/monitoring/cmk.tf

```diff
@@ -0,0 +1,65 @@
+# Creates/manages KMS CMK
+resource "aws_kms_key" "cmk" {
+  count = var.enable_cmk ? 1 : 0
+
+  #description             = var.description
+  customer_master_key_spec = var.key_spec
+  is_enabled               = var.enabled
+  enable_key_rotation      = var.rotation_enabled
+  #tags                    = var.tags
+  deletion_window_in_days = 30
+
+  policy = jsonencode({
+    "Version" : "2012-10-17",
+    "Id" : "key-default-1",
+    "Statement" : [
+      {
+        "Sid" : "Enable IAM User Permissions",
+        "Effect" : "Allow",
+        "Principal" : {
+          "AWS" : "arn:aws:iam::${data.aws_caller_identity.current.account_id}:root"
+        },
+        "Action" : "kms:*",
+        "Resource" : "*"
+      },
+      {
+        "Sid" : "Allow access for Key Administrators",
+        "Effect" : "Allow",
+        "Principal" : {
+          #Use 'var.sns_key_admin_arn' if available and root if not provided
```

Comment:
leave empty space between the # and the Use word

Reply:
done.